packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Products/__tests__/__snapshots__/products.spec.js.snap

```diff
@@ -1,84 +1,11 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`render products with all props & no products 1`] = `
-<div
-  className=" test-class"
-  style={
-    Object {
-      "border": "solid",
-      "borderColor": "red",
-      "borderRadius": "15px",
-      "borderWidth": "10px",
-      "marginBottom": "10px",
-      "marginLeft": "10px",
-      "marginRight": "10px",
-      "marginTop": "10px",
-      "paddingBottom": "10px",
-      "paddingLeft": "10px",
-      "paddingRight": "10px",
-      "paddingTop": "10px",
-      "textAlign": "right",
-    }
-  }
->
-  <div>
-    No products to display
-  </div>
-</div>
-`;
+exports[`render products with all props & no products 1`] = `null`;
```

Comment:
I don't think this is something to fix in this PR but is @davemacaulay and team aware that these snaps are now incorrect?